File: Input_and_Output_coding/Input_and_Output.py
name = input("Enter your name: ")
print("Hello", name)

age = int(input("Enter your age: "))

# F-strings (recommended)
print(f"My name is {name} and I am {age} years old")

# Format method
print("Hello {} and welcome!".format(name))

# Old style formatting
print("Name: %s, Age: %d" % (name, age))

# check data type
print(type(name))
print(type(age))

File: Input_and_Output_coding/Input_and_Output.js

let name = prompt("Enter your name:");
console.log("Hello", name);

let age = parseInt(prompt("Enter your age:"), 10);

console.log(`My name is ${name} and I am ${age} years old`);

console.log("Name: " + name + ", Age: " + age);

//  Type check
console.log(typeof name); // string
console.log(typeof(age));  // number
